CHANGELOG.md

```diff
@@ -70,9 +84,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Logging and Analytics support added.
 - End-to-end testing using Minikube and Travis-ci.
 
-[Unreleased]: https://github.com/kadalu/kadalu/compare/0.5.0...HEAD
+[Unreleased]: https://github.com/kadalu/kadalu/compare/0.6.0...HEAD
 [0.1.0]: https://github.com/kadalu/kadalu/compare/e434f25...0.1.0
 [0.2.0]: https://github.com/kadalu/kadalu/compare/0.1.0...0.2.0
 [0.3.0]: https://github.com/kadalu/kadalu/compare/0.2.0...0.3.0
 [0.4.0]: https://github.com/kadalu/kadalu/compare/0.3.0...0.4.0
 [0.5.0]: https://github.com/kadalu/kadalu/compare/0.4.0...0.5.0
+[0.6.0]: https://github.com/kadalu/kadalu/compare/0.5.0...0.6.0
```

Comment:
just a thought, should we change the order of this list? ie, latest one at the top ? For now it is OK.

Reply:
This section is just the reference. Actual link is on the top. See this page https://github.com/kadalu/kadalu/blob/master/CHANGELOG.md